tests/microos/toolbox.pm

```diff
@@ -76,25 +86,28 @@ sub run {
     # Back to root
     select_console 'root-console';
 
-    record_info 'Test', 'Pulling toolbox image from different registry';
-    # Switch default registries for openSUSE MicroOS and SLE Micro
-    if (is_sle_micro) {
-        assert_script_run 'echo -e "REGISTRY=registry.opensuse.org\nIMAGE=opensuse/toolbox" > ~/.toolboxrc';
-        validate_script_output 'toolbox -r cat /etc/os-release', sub { m/opensuse/ }, timeout => 180;
-    } else {
-        assert_script_run 'echo -e "REGISTRY=registry.suse.com\nIMAGE=suse/sle-micro/5.0/toolbox" > ~/.toolboxrc';
-        validate_script_output 'toolbox -r cat /etc/os-release', sub { m/sles/ }, timeout => 180;
+    unless ($toolbox_image_to_test) {
```

Comment:
Why are we limiting the toolbox tests to the cases when we have a custom toolbox URI specified? Is toolbox not being tested by default also for SLEM?

Reply:
Yes, this module is executed in regular SLEM container jobs, but using the default image which comes from registry.suse.com.

I put this `unless` here to skip this checks which are testing functionality of `toolboxrc` using different parameters, which is what we need to test the toolbox image under test..

Reply:
Ahh, I see. The actual tests are coming later. Thnx for the clarification!